src/theme/index.ts

```diff
@@ -47,35 +50,13 @@ const theme = {
   colors,
   colorsDeprecated,
   fonts,
+  others,
```

Comment:
others ???

Reply:
Its all tokens that can't be in normal behavior, like overlay background that is not part of normal tokens and should be separated to "specific" category.

If it sounds strange we can put this overlay into `colors.overlay` right away 😄

Reply:
yeaaaaah but the theme does not only represents colors, the `other` or `extra` key at root is the beginning of a big mess 😄 

These are still colors ? Maybe colors.others ?

Reply:
From now it is still colors, so I think I can still put in into `colors.overlay` it will make sense

Reply:
@chambo-e I did a little fix of script + this change right here: https://github.com/scaleway/scaleway-ui/pull/1259